docs/using/emr.md

```diff
@@ -7,4 +7,124 @@ nav_order: 6
+   Each step using lakeFS should pass the appropriate config params.
+
+## Configuration on cluster creation 
+Use the below configuration when creating the cluster. You may delete any app configuration which is irrelavant for your use-case.
+ ```json
+[
+  {
+    "Classification": "presto-connector-hive",
+    "Properties": {
+      "hive.s3.aws-secret-key": "$LAKEFS_SECRET_KEY",
+      "hive.s3.aws-access-key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "hive.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "hive.s3-file-system-type": "PRESTO"
+    }
+  },
+  {
+    "Classification": "hive-site",
+    "Properties": {
+      "fs.s3.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "fs.s3a.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3a.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3a.endpoint": "$LAKEFS_ENDPOINT"
+    }
+  },
+  {
+    "Classification": "hdfs-site",
+    "Properties": {
+      "fs.s3.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "fs.s3a.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3a.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3a.endpoint": "$LAKEFS_ENDPOINT"
+    }
+  },
+  {
+    "Classification": "core-site",
+    "Properties": {
+      "fs.s3.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "fs.s3a.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3a.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3a.endpoint": "$LAKEFS_ENDPOINT"
+    }
+  },
+  {
+    "Classification": "emrfs-site",
+    "Properties": {
+      "fs.s3.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "fs.s3a.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3a.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3a.endpoint": "$LAKEFS_ENDPOINT"
+    }
+  },
+  {
+    "Classification": "mapred-site",
+    "Properties": {
+      "fs.s3.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3.endpoint": "$LAKEFS_ENDPOINT",
+      "fs.s3a.access.key": "$LAKEFS_SECRET_ACCESS_KEY",
+      "fs.s3a.secret.key": "$LAKEFS_SECRET_KEY",
+      "fs.s3a.endpoint": "$LAKEFS_ENDPOINT"
+    }
+  },
+  {
+    "Classification": "spark-defaults",
+    "Properties": {
+      "spark.sql.catalogImplementation": "hive"
+    }
+  }
+]
+
+```
+
+## Configuration on adding a step
+When a cluster was created without the above configuration, you can still use lakeFS when adding a step.
+
+For example, when creating a Spark job: 
+
+```shell
+aws emr add-steps --cluster-id j-197B3AEGQ9XE4 \
+--steps="Type=Spark,Name=SparkApplication,ActionOnFailure=CONTINUE,\
+Args=[--conf,spark.hadoop.fs.s3a.access.key=AKIAIOSFODNN7EXAMPLE,\
```

Comment:
example values or variables? whatever you choose, it should probably be consistent across the guide..

Reply:
Done.